nmigen_soc/wishbone/bus.py

```diff
@@ -1,12 +1,13 @@
 from enum import Enum
 from nmigen import *
+from nmigen.lib.coding import Decoder as SignalDecoder
```

Comment:
`SignalDecoder` seems ambiguous. How about `from nmigen.lib import coding`, and then use `coding.Decoder` ?

Reply:
Done